cmd/cluster-agent/api/agent/agent.go

```diff
@@ -122,12 +124,19 @@ func getCheckLatestEvents(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
-// TODO: complete it
-func getContainerMetadata(w http.ResponseWriter, r *http.Request) {
+func getPodMetadata(w http.ResponseWriter, r *http.Request) {
 	vars := mux.Vars(r)
-	host := vars["host"]
-	cID := vars["container"]
-	// TODO: check host (in store/ not in store)
-	// then check container
-	w.Write([]byte("host: " + host + ", container: " + cID))
+	nodeName := vars["nodeName"]
+	podName := vars["podName"]
+	svcList := as.GetPodSvcs(nodeName, podName)
+
+	slcB, err := json.Marshal(svcList)
+	if err != nil {
+		log.Errorf("Could not process the list of services of: %s", podName)
+	}
+	if len(svcList) != 0 {
+		w.Write(slcB)
+		return
+	}
```

Comment:
What do you think of adding an adapted http status code like:
```go
w.WriteHeader(404)
```

Reply:
yeah, nice touch. I'll add 200 if the len(svclist)> 0